RecoTracker/LSTCore/standalone/README.md

```diff
@@ -1,53 +1,100 @@
-# TrackLooper
+# How to set up standalone LST
 
-
-## Quick Start
-
-
-### Setting up LSTPerformanceWeb (only for lnx7188 and lnx4555)
+## Setting up LSTPerformanceWeb (only for lnx7188 and lnx4555)
 
 For lnx7188 and lnx4555 this needs to be done once
 
     cd /cdat/tem/${USER}/
     git clone git@github.com:SegmentLinking/LSTPerformanceWeb.git
 
-### Setting up container (only for lnx7188)
+## Setting up container (only for lnx7188)
 
 For lnx7188 this needs to be done before compiling or running the code:
 
     singularity shell --nv --bind /mnt/data1:/data --bind /data2/segmentlinking/ --bind /opt --bind /nfs --bind /mnt --bind /usr/local/cuda/bin/ --bind /cvmfs  /cvmfs/unpacked.cern.ch/registry.hub.docker.com/cmssw/el8:x86_64
 
-### Setting up the code
+## Setting up LST
+
+There are two way to set up LST as a standalone, either by setting up a full CMSSW area, which provides a unified setup for standalone and CMSSW tests, or by `sparse-checkout` only the relevant package and using them independent of CMSSW. A CVMFS-less setup is also provided for the second option.
+
+### Setting up LST within CMSSW (preferred option)
+
+```bash
+CMSSW_VERSION=CMSSW_14_2_0_pre4 # Change with latest/preferred CMSSW version
+LST_BRANCH=CMSSW_14_1_0_pre3_LST_X_LSTCore_realfiles_batch1_devel # Change to the appropriate branch
+cmsrel ${CMSSW_VERSION}
+cd ${CMSSW_VERSION}/src/
+cmsenv
+git cms-init
+git remote add SegLink git@github.com:SegmentLinking/cmssw.git
+git fetch SegLink ${LST_BRANCH}
+git checkout ${LST_BRANCH}
+git cms-addpkg RecoTracker/LST RecoTracker/LSTCore
+scram b -j 12
+cd RecoTracker/LSTCore/standalone
+```
+
+The data files for LST will be fetched from CVMFS. However, if new data files are needed, the need to be manually placed (under `$CMSSW_BASE/external/$SCRAM_ARCH/data/RecoTracker/LSTCore/data/`). This is done by running:
+
+```bash
+mkdir -p $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoTracker/LSTCore/
+cd $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoTracker/LSTCore/
+git clone git@github.com:cms-data/RecoTracker-LSTCore.git data
+<modify the needed files>
+cd -
+```
+
+### Setting up LST outside of CMSSW
+
+For this setup, dependencies are still provided from CMSSW through CVMFS but no CMSSW area is setup. This is done by running the following commands.
+
+``` bash
+LST_BRANCH=CMSSW_14_1_0_pre3_LST_X_LSTCore_realfiles # Change to the development branch
```

Comment:
Maybe update this to at least be consistent with the one mention in the section just above?